.github/workflows/archive.yml

```
@@ -0,0 +1,76 @@
+#
+# This source file is part of the Stanford Biodesign Digital Health Group open-source organization
+#
+# SPDX-FileCopyrightText: 2022 Stanford University and the project authors (see CONTRIBUTORS.md)
+#
+# SPDX-License-Identifier: MIT
+#
+
+name: Build XCArchive
+
+on:
+  workflow_call:
+    inputs:
+      workspaceFile:
+        description: 'The path of the Xcode Workspace file (including the file extension).'
+        type: string
+        required: true
+      XCArchiveName:
```

Would suggest to keep inputs lowercase:
```suggestion
      xcarchivename:
```